build/Rulesets/Roslyn_BuildRules.ruleset

```diff
@@ -123,4 +123,7 @@
   <Rules AnalyzerId="XmlDocumentationComments.Analyzers" RuleNamespace="XmlDocumentationComments.Analyzers">
     <Rule Id="RS0010" Action="Warning" />
   </Rules>
+  <Rules AnalyzerId="Microsoft.NetCore.Analyzers" RuleNamespace="Microsoft.NetCore.Analyzers">
+    <Rule Id="CA9999" Action="None" /> <!-- We know the analyzers will fail during a bootstrap build -->
```

Comment:
One fear I had with this is that we could get into the following situation:

- Analyzers flags a valid problem in our code. 
- Jenkins is green because we aren't running analyzers.
- VSTS fails because it doesn't bootstrap hence uses analyzers and fails the build.

I checked though and microbuild in Jenkins also doesn't bootstrap. Hence we're covered here. 

Mostly wanted to explain that so others understand all the different ways in which we buil here.